### DISCUSSION TITLE
Equipment requirements

### DISCUSSION THREAD

**Discussion by janstehno** [janstehno/cotw-companion#42] [Ideas]
As title says, there would be an information about how much weapon points you need to unlock certain weapons and what level you need to unlock certain callers.

[original discussion](https://github.com/janstehno/cotwcompanion/discussions/34)

**Comment by JeanRoxus** [comment]
Is it possible to know the power or noise of each weapon? I mean, the vasquez cyclone seems to have many virtues, but when using it, a good shot usually ends in a long crawl or the piece never bleeding to death. (with some exceptions)

**Reply by janstehno** [reply]
I am not sure if I understand your question. A lot of weapons scare animals within 250-300 meters I think. There are exceptions like Virant 22LR for example. If we would talk about power, I would say it is determined by penetration stat on its ammo actually. The more penetration the bigger possibility of the animal going down. There is also an accuracy stat for each weapon, so missing with something that has lower accuracy is pretty much expected at long range.

**Comment by janstehno** [comment]
Added in 1.9.0
